about.md

```diff
@@ -1,11 +1,74 @@
 ---
 layout: page
-title: About
+title: About the GeoBlacklight Community
 permalink: /about/
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+- [Community Structure](#community-structure)
+- [Technical Values](#technical-values)
+- [Project Roadmap](#project-roadmap)
 
-You can find the source code for the Jekyll new theme at: [github.com/jglovier/jekyll-new](https://github.com/jglovier/jekyll-new)
+GeoBlacklight is a multi-institutional, open source geospatial data discovery software application. Based on the Blacklight Project, GeoBlacklight began in 2014 and has reached its third major release in 2020. GeoBlacklight has been adopted as an interface for geospatial data discovery by over 25 academic libraries and cultural heritage institutions.
 
-You can find the source code for Jekyll at [github.com/jekyll/jekyll](https://github.com/jekyll/jekyll)
+### Community Structure
```

Comment:
Should we break this out into a separate pull request? That it may not hold up moving forward with the about page and other parts that have been more broadly discussed?

Reply:
Do you mean we should include the Technical Values list, but not the Community Structure section?  If so, perhaps we can move the current community structure to a blog post that announces our intention to address the topic. The About page could have a named placeholder and a link to the blog post for more information about our progress.

Reply:
Yep, and just the about page in general. My thought here is that most definitely we should define the community structure, but before that has been figured all out, can we ship the about page. I would imagine it might be a sometime before we have all of the details in place about the community structure.